### DISCUSSION TITLE
The query uses a row limiting operator ('Skip'/'Take') without an 'OrderBy' operator. This may lead to unpredictable results.

### DISCUSSION THREAD

**Discussion by fratting** [alirezanet/Gridify#109] [General]
I don't have any cases of explicit .Skip/.Take/.First/.Single with an OrderBy(). Could this be from how Gridify is applying paginated queries? Thanks.

**Comment by alirezanet** [comment]
Hi @fratting, 

Yes, if you are using the `Gridify` method by default it will limit the result (20 items) and uses the `GridifyQuery`in order to change the page number and total item count. 
if you don't want the paging and ordering functionalities, you can use for example the `ApplyFiltering` extension method. this extension only applies filtering to your query.
